nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -59,81 +62,101 @@ class MinMaxQuantizationParameters(AlgorithmParameters):
                                       per_channel=False)
 
     def __init__(self,
+                 number_samples: int = 300,
                  preset: QuantizationPreset = QuantizationPreset.PERFORMANCE,
-                 weight_bits: int = 8,
-                 weight_granularity: Granularity = Granularity.PERCHANNEL,
-                 activation_bits: int = 8,
-                 activation_granularity: Granularity = Granularity.PERTENSOR,
-                 number_samples: int = 100,
-                 target_device: HWConfigType = HWConfigType.CPU,
-                 range_type: RangeType = RangeType.MEAN_MINMAX,
+                 weight_bits: Optional[int] = None,
+                 weight_granularity: Optional[Granularity] = None,
+                 weight_signedness_to_force: Optional[bool] = None,
+                 activation_bits: Optional[int] = None,
+                 activation_granularity: Optional[Granularity] = None,
+                 activation_signedness_to_force: Optional[bool] = None,
+                 target_device: TargetDevice = TargetDevice.ANY,
+                 range_type: Optional[RangeType] = None,
                  quantize_outputs: bool = False,
-                 ignored_scopes: List[str] = None,
+                 ignored_scopes: Optional[List[str]] = None,
                  ):
         """
-        :param weight_quantizer_config: QuantizerConfig instance for the weights.
-        :param activation_quantizer_config: QuantizerConfig instance for the activations.
         :param number_samples: Number of samples for the statistics collection.
+        :param preset: Preset parameter for Quantization.
+            Defines the mode: symmetric or asymmetric of the activation quantizers.
+        :param weight_bits: Bitwidth for the weight quantizers.
+        :param weight_granularity: Type of quantization granularity for weight quantizers.
+            Could be per-channel or per-tensor.
+        :param weight_signedness_to_force: Defines whether the datatype of the weight quantizers should be forced.
```

Comment:
Sounds a bit crazy. I would expect something like `force_signed_weigths`,  `force_signed_activations`, `force_` can be omitted even.

Reply:
```
weight_signedness_to_force -> signed_weights
activation_signedness_to_force -> signed_activations
```